Essentials/src/com/earth2me/essentials/commands/Commanddelkit.java

```diff
@@ -0,0 +1,46 @@
+package com.earth2me.essentials.commands;
+
+import com.earth2me.essentials.User;
+
+import org.bukkit.Server;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static com.earth2me.essentials.I18n.tl;
+
+public class Commanddelkit extends EssentialsCommand {
+
+    public Commanddelkit() {
+        super("deletekit");
+    }
+
+    @Override
+    public void run(final Server server, final User user, final String commandLabel, final String[] args) throws Exception {
+        if (args.length != 1) {
+            throw new NotEnoughArgumentsException();
+        }
+
+        String kitname = args[0];
+        if (!ess.getSettings().isPastebinCreateKit()) {
```

Comment:
Again, just delete the kit. There's no reason to check `pastebin-createkit` here.

Reply:
There may be a problem with this. If do not specify the pastebin-createkit check and simply delete it from the file, an NPE error may occur in the console when the kit with the given name does not exist. So we need to check the kit to see if it exists or not.